cardano-sl.cabal

```diff
@@ -120,6 +120,7 @@ library
                         Pos.Util.Arbitrary
                         Pos.Util.BackupPhrase
                         Pos.Util.JsonLog
+                        Pos.Util.Servant
```

Comment:
Ok, fixed.
I just discovered that I'd broken build with `--no-wallet` before. Will deal with it tomorrow